storage/pkg/flagset/driverlocal.go

```diff
@@ -2,17 +2,17 @@ package flagset
 
 import (
 	"github.com/micro/cli/v2"
-	"github.com/owncloud/ocis/ocis-reva/pkg/config"
+	"github.com/owncloud/ocis/storage/pkg/config"
 )
 
 // DriverLocalWithConfig applies cfg to the root flagset
 func DriverLocalWithConfig(cfg *config.Config) []cli.Flag {
 	return []cli.Flag{
 		&cli.StringFlag{
 			Name:        "storage-local-root",
-			Value:       "/var/tmp/reva/root",
+			Value:       "/var/tmp/ocis/local",
```

Comment:
:ok_hand: the local driver itself currently serves no special purpose, so it can be named after the driver.